pacifica/cli/methods.py

```diff
@@ -146,12 +146,17 @@ def generate_requests_auth(global_ini):
             )
         }
     elif auth_type == 'basic':
+        username = getenv('AUTHENTICATION_USERNAME', global_ini.get('authentication', 'username'))
```

Comment:
You introduce a few new local variables, but only reference them once. Call the function in the same line instead of referencing the variable.

Reply:
Done.